bin/install

```diff
@@ -49,10 +49,10 @@ install_lua() {
         # If we are installing Lua 5.x or greater install LuaRocks as well
         if version_5x_or_greater $version; then
             echo "Installing LuaRocks..."
-            curl https://keplerproject.github.io/luarocks/releases/luarocks-2.4.1.tar.gz --output luarocks.tar.gz || exit 1
+            curl -L 'https://luarocks.org/releases/luarocks-3.1.3.tar.gz' --output luarocks.tar.gz || exit 1
             tar zxpf luarocks.tar.gz || exit 1
-            cd luarocks-2.4.1 || exit 1
-            ./configure --with-lua=$install_path --prefix=$install_path/luarocks || exit 1
+            cd luarocks-3.1.3 || exit 1
+            ./configure --with-lua=$install_path --with-lua-include=$install_path/include --with-lua-lib=$install_path/lib --prefix=$install_path/luarocks || exit 1
```

Comment:
Do we need `--with-lua-include=$install_path/include` on this line? I think it was working before without this.

Reply:
Yeah it doesn't work otherwise. Gives LUA_INCDIR not found. This is because a standard install of Lua puts include files in `include/lua` and we are just putting them in `include`. [This is the part of luarocks that checks for include headers](https://github.com/luarocks/luarocks/blob/3e12f3df50e4f6f0035f13b255d380fbf207a7c1/configure#L429). I think the best solution is to change the way we install Lua to use the standard directories. If you are fine with that I can open a pull request.